FWCore/ParameterSet/python/MassReplace.py

```diff
@@ -1,5 +1,19 @@
 from __future__ import print_function
 import FWCore.ParameterSet.Config as cms
+from FWCore.ParameterSet.Types import _ProxyParameter
+import copy
+
+def isinstanceWproxy(value, typeId):
```

Comment:
I think this is asking the wrong question. Checking for `isinstance` in python is usually a poor choice because the language uses _duck typing_ . (Yes, we have `isinstance` in many places in our code for historic reasons and because I'm a poor python programmer :) ). I don't think we want to as an `isinstance` question, instead we want to extend our parameter type classes with a `canAssignUsing` method. Then anything that can be assigned from  a `cms.InputTag` would work properly.

Reply:
OK, please let me know if ```canAssignUsing``` can be added to our python Types soon.
Will it be something that can be backported to 11_1_X?